### PR TITLE
[MNOE-282] App comparison config

### PR DESCRIPTION
**Pull request by c3gdlk**
…unning_instances_count into the API response

### CODE REVIEW

File: core/lib/generators/mno_enterprise/install/templates/config/settings.yml [hunk 1]
```diff
@@ -24,3 +24,7 @@ developer:
 # Enable Reviews in the marketplace
 reviews:
   enabled: false
+
+marketplace:
+  comparison:
```
Comment (x4d3) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: I would prefer apps_comparison

Comment (c3gdlk) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: @x4d3 this name came from the issue description. Please confirm that I need to use the `apps_comparison`, it's 5 min to change

Comment (x4d3) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: ok I did not see it. Ignore my remark then.

Comment (ouranos) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: All good. I'll default it to `false` though.

Comment (ouranos) [inline core/lib/generators/mno_enterprise/install/templates/config/settings.yml]: Done in #201

**Pull request merged**